05.wc/wc.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+LINES_WIDTH = 7
+WORDS_AND_CHARACTERS_WIDTH = 8
+
+def main
+  options = parse_options
+
+  if ARGV.empty?
+    print_statistics_from_standard_input(options)
+  else
+    print_statistics_and_total_from_argv(options)
+  end
+end
+
+def parse_options
+  options = {}
+
+  OptionParser.new do |opt|
+    opt.on('-l') { options[:lines] = true }
+    opt.on('-w') { options[:words] = true }
+    opt.on('-c') { options[:characters] = true }
+    opt.parse!(ARGV)
+  end
+
+  options = {lines: true, words: true, characters: true} if options.empty?
+  options
```

Comment:
```suggestion
  options.empty? ? {lines: true, words: true, characters: true} : options
```

とすれば1行になるし変数への再代入も避けられます。

[プログラミング初心者は変数の再代入をなるべく避けよう \| FBC](https://bootcamp.fjord.jp/pages/avoid-re-assign)

あと、`{ a: 1 }`か`{a: 1}`か、スタイルを統一した方がいいですね。

Reply:
ありがとうございます！
変数の再代入を避ける意識がついてれば「分岐できないか？」という考えには至っているはずですね…（ご提案いただいたコードほどきれいに書けるかはわかりませんがｗ）
しばらくは重点チェック事項にしたいと思います。

ハッシュのスペースの有無は、前回`rubocop`を通したあとにコードを触ってしまったようです（＋最終的なセルフチェックの甘さ）。
以後気を付けます😿